src/software/ai/navigator/path_planner/theta_star_path_planner.h

```diff
@@ -121,73 +132,72 @@ class ThetaStarPathPlanner : public PathPlanner
 
        private:
         Coordinate parent_;
-        double f_, g_;
+        double path_cost_and_end_dist_heuristic_;
+        double best_path_cost_;
         bool initialized_;
     };
 
     /**
      * Returns if a cell is within bounds of grid
      *
-     * @param test_coord Coordinate to consider
+     * @param coord Coordinate to consider
      *
      * @return true if cell is valid
      */
-    bool isCoordValid(Coordinate test_coord);
+    bool isCoordValid(const Coordinate &coord) const;
```

Comment:
similar comment here, not blocking, but `isCoordInNavigableRegion` might be a bit more descriptive